scrapy/settings/default_settings.py

```diff
@@ -230,9 +230,12 @@
 STATS_DUMP = True
 
 STATSMAILER_RCPTS = []
-
-TEMPLATES_DIR = abspath(join(dirname(__file__), '..', 'templates'))
-
+TEMPLATES_DIR_BASE = [
```

Comment:
I looked into ways to do this. My idea would be to use os.getcwd() to get the directory where the script is running and add this to the user's version of scrapy.cfg (append at the end or something). Is this what you had in mind?